src/request-item/request-item.tsx

```diff
@@ -9,14 +9,15 @@ export type RequestItemProps = h.JSX.HTMLAttributes<HTMLDivElement> & {
 	requestReplies: number
 	requestDate: Date
 	requestAmt: number
-	myRequest?: boolean
+	myself?: boolean
 	avatar?: string
 	detail?: boolean
 	requesterDetail?: boolean
```

Comment:
Is this necessary? I think it was left over from when I wasn't sure how to pass props into the child component, I must have forgotten to remove it.